R/Spectra.R

```diff
@@ -2639,3 +2642,15 @@ setMethod("combinePeaks", "Spectra", function(object, tolerance = 0, ppm = 20,
         ppm, " and tolerance = ", tolerance, ".")
     object
 })
+
+
+#' @rdname Spectra
+#'
+#' @importFrom MsCoreUtils entropy
+setMethod("entropy", "Spectra", function(object, normalized = TRUE) {
+  if (length(object))
```

Comment:
for `normalize == TRUE` we would need to call `nentropy`, for `normalized = FALSE` `entropy` - I would suggest to add something like:

```r
if (normalized) entropy_fun <- nentropy
else entropy_fun <- entropy
unlist(.peaksapply(
    object, FUN = function(pks, ...) entropy_fun(pks[, "intensity"])), 
    use.names = FALSE
))
```

ah, and please remove the `na.rm = TRUE` since the `entropy` function does not support that.